### PR TITLE
[Docs / Wysiwyg fields] Fix wysiwyg editor example config

### PR DESCRIPTION
**Pull request by BlackbitDevs**
Since Pimcore 10 the example editor config in https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Text_Types.html#page_Editor-Configuration does not work anymore. The reason is that in https://github.com/pimcore/pimcore/blob/1be12b0493bc9153783c8d85210f9863637aec3c/bundles/AdminBundle/Resources/public/js/pimcore/object/tags/wysiwyg.js#L193 `Ext.decode()` gets used - but the difference with Pimcore 10 is that in this function
```javascript
try {
            // check USE_NATIVE_JSON here so it can be changed if needed
            if (hasNative && Ext.USE_NATIVE_JSON) {
                return JSON.parse(json);
            }
            return doDecode(json);
        } catch (e) {
            if (safe) {
                return null;
            }
            Ext.raise({
                sourceClass: "Ext.JSON",
                sourceMethod: "decode",
                msg: "You're trying to decode an invalid JSON String: " + json
            });
        }
```

`Ext.USE_NATIVE_JSON` is `true` whereas in Pimcore 6 it was `false`. Thus in Pimcore 6 the `doDecode()` function was used which is much more tolerant. The native `JSON.parse()` is really picky. Back to the docs example:
```markdown
{
  toolbarGroups : [ { name: 'links' }],
  enterMode: CKEDITOR.CKEDITOR.ENTER_BR,
}
```
This does not work for the following reasons:
- JSON attributes need to be quoted
- trailing comma after `enterMode` not allowed
- `CKEDITOR.CKEDITOR.ENTER_BR` got renamed to `CKEDITOR.ENTER_BR`
- constants cannot be resolved(!)

Especially the last point is really bad because you cannot use the generated configurations from [the linked CKEditor configuration docs](https://ckeditor.com/docs/ckeditor4/latest/features/toolbar.html) or the [CKEditor configurator](https://ckeditor.com/latest/samples/toolbarconfigurator). Because of this this PR does not only fix the docs but disables `Ext.USE_NATIVE_JSON` for this.

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/object/tags/wysiwyg.js [hunk 1]
```diff
@@ -190,7 +190,10 @@ pimcore.object.tags.wysiwyg = Class.create(pimcore.object.tags.abstract, {
         }
 
         if(this.fieldConfig.toolbarConfig) {
+            var useNativeJson = Ext.USE_NATIVE_JSON;
```
Comment (robertSt7) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/tags/wysiwyg.js]:
```suggestion
            const useNativeJson = Ext.USE_NATIVE_JSON;
```

**Review by robertSt7 — CHANGES_REQUESTED** [review]
Could you also please merge 10.5 in your branch to fix the tests? Thank you

**Comment by BlackbitDevs** [comment]
@robertSt7 Done.

**Comment by robertSt7** [comment]
@BlackbitNeueMedien Thanks a lot

**Pull request merged**